packages/prettier-config/package.json

```diff
@@ -9,7 +9,8 @@
     "index.json"
   ],
   "dependencies": {
-    "prettier": "2.4.1"
+    "@trivago/prettier-plugin-sort-imports": "^4.3.0",
+    "prettier": "3.2.5"
```

Comment:
From prettier docs:

> Note: If you forget to install Prettier first, npx will temporarily download the latest version. That’s not a good idea when using Prettier, because we change how code is formatted in each release! It’s important to have a locked down version of Prettier in your package.json.

I believe this applies to dependency versions as well. It's better to keep it locked to keep code consistent. Every other project will pull in our `prettier-config` which will constrain the prettier version. Other packages include a relaxed dependency version just so prettier is installed at all, I had some issues where if I didn't add it explicitly.